swap/src/bin/swap_cli.rs

```diff
@@ -136,19 +151,14 @@ async fn main() -> Result<()> {
                 );
             }
 
-            let send_bitcoin = bitcoin_wallet.max_giveable(TxLock::script_size()).await?;
+            let max_giveable = bitcoin_wallet.max_giveable(TxLock::script_size()).await?;
+            let max_accepted = bid_quote.max_quantity;
 
-            let (event_loop, event_loop_handle) = EventLoop::new(
-                &seed.derive_libp2p_identity(),
-                alice_peer_id,
-                alice_addr,
-                bitcoin_wallet.clone(),
-            )?;
-            let handle = tokio::spawn(event_loop.run());
+            let send_bitcoin = min(max_giveable, max_accepted);
```

Comment:
We default to the `max_accepted` in case the user transfers more - I like that, because this solves the issue with some faucets not allowing to enter an amount!
I assume the behavior when starting it after a swap would be that it immediately triggers another swap with the remaining balance?.

Reply:
Yes, that would be the behaviour.